src/toil/common.py

```diff
@@ -691,9 +691,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             if (exc_type is not None and self.config.clean == "onError" or
                             exc_type is None and self.config.clean == "onSuccess" or
                         self.config.clean == "always"):
-                logger.info("Attempting to delete the job store")
-                self._jobStore.destroy()
-                logger.info("Successfully deleted the job store")
+                try:
+                    self._jobStore.destroy()
+                    logger.info("Successfully deleted the job store: %s" % str(self._jobStore))
+                except:
+                    logger.info("Failed to delete the job store: %s" % str(self._jobStore))
```

Comment:
Exception should be re-raised here after logging. As it is this will swallow the exception and just log without crashing the program, which i don't think is what we want

Reply:
Fixed.